src/scripts/components/VideoOverlay.js

```diff
@@ -150,10 +149,7 @@ const ProfileButtonWrapper = styled.div`
 `
 
 const CentralizedContent = styled.div`
-  align-items: center;
-  display: flex;
-  height: 100%;
-  justify-content: center;
+  ${FlexCenterStyle} height: 100%;
```

Comment:
If we put these style variables at the end of our style blocks perhaps our auto-formatter won't f*** with the positioning as much?

Reply:
I don't know for sure. Mostly of our problems happens when we have to use `:hover` selector.